src/status_im/ui2/screens/chat/messages/message.cljs

```diff
@@ -506,166 +323,65 @@
-                                                  :on-press #(pin-message message)
-                                                  :label    (i18n/label (if pinned :t/unpin-from-chat :t/pin-to-chat))
-                                                  :icon     :i/pin
-                                                  :id       (if pinned :unpin :pin)}
-                                                 {:type     :danger
-                                                  :on-press #(re-frame/dispatch
-                                                              [:chat.ui/delete-message-for-me message
-                                                               config/delete-message-for-me-undo-time-limit-ms])
-                                                  :label    (i18n/label :t/delete-for-me)
-                                                  :icon     :i/delete
-                                                  :id       :delete-for-me}
-                                                 (when (and outgoing config/delete-message-enabled?)
-                                                   {:type     :danger
-                                                    :on-press #(re-frame/dispatch [:chat.ui/delete-message
-                                                                                   message
-                                                                                   config/delete-message-undo-time-limit-ms])
-                                                    :label    (i18n/label :t/delete-for-everyone)
-                                                    :icon     :i/delete
-                                                    :id       :delete})]))]
-        (reset! ref on-long-press)
-        [message-content-wrapper message
-         [rn/touchable-opacity
-          (when-not modal
-            {:on-long-press on-long-press
-             :on-press      (fn []
-                              (reset! show-timestamp? true))})
-          [rn/view style/message-view-wrapper
-           [message-timestamp message show-timestamp?]
-           [rn/view {:style (style/message-view message) :accessibility-label :audio-message}
-            [rn/view {:style (style/message-view-content)}
-             [message.audio/message-content message] [message-status message]]]]]
-         reaction-picker]))))
+;; EMOJI
+(defn emoji []
+  (fn [{:keys [content] :as message}]
+    [rn/view style/message-view-wrapper
+     [rn/view (style/message-view message)
+      [rn/view {:style (style/message-view-content)}
+       [rn/view {:style (style/style-message-text)}
+        [rn/text {:style (style/emoji-message message)}
+         (:text content)]]]]]))
+
+;; STICKER
+(defn sticker
+  [{:keys [content]}]
+  [fast-image/fast-image {:style  {:margin 10 :width 140 :height 140}
+                          :source {:uri (str (-> content :sticker :url) "&download=true")}}])
+
+;;IMAGE
+(defn message-content-image
```

Comment:
@flexsurfer, can you shed some light about why this PR changes the old `chat/messages/message.cljs` file instead of just removing stuff and moving to the new namespaces?

Reply:
because it's not implemented yet, and should be implemented from scratch, we have a bunch of issues for that, but we want to keep messages rendering, that's why we can't just remove them

Reply:
I see now, thank!